test/unit/org/apache/cassandra/hints/HintsStoreTest.java

```diff
@@ -137,6 +138,26 @@ public void testConcurrentDeleteExpiredHints() throws Exception {
         assertFalse("All hints files should be deleted", store.hasFiles());
     }
 
+    @Test
+    public void testPendingHintsInfo() throws Exception
+    {
+        HintsStore store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
+        assertEquals(new PendingHintsInfo(store.hostId, 0, Long.MAX_VALUE, Long.MIN_VALUE),
+                     store.getPendingHintsInfo());
+
+        final long t1 = System.currentTimeMillis();
+        writeHints(directory, new HintsDescriptor(hostId, t1), 100, t1);
+        store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
+        assertEquals(new PendingHintsInfo(store.hostId, 1, t1, t1),
+                     store.getPendingHintsInfo());
+        final long t2 = t1 + 1;
+        writeHints(directory, new HintsDescriptor(hostId, t2), 100, t2);
+        store = HintsCatalog.load(directory, ImmutableMap.of()).get(hostId);
+        assertEquals(new PendingHintsInfo(store.hostId, 2, t1, t2),
+                     store.getPendingHintsInfo());
+
```

Comment:
nit: unnecessary empty line

Reply:
Fixed in 6f5f1e22cb23237bb07e76a5e38b20d9965dcf00.